pstress/pstress-run.sh

```diff
@@ -908,7 +908,7 @@ pstress_test(){
         create_local_config component_keyring_vault
       fi
     fi
-    MYEXTRA=
+    MYEXTRA=${MYEXTRA}
```

Comment:
This change looks good to me, but it will require code fixes in other areas too. 
For ex. consider this if condition:

`    if [ ${ADD_RANDOM_OPTIONS} -eq 1 ]; then  # Add random mysqld --options to MYEXTRA
      OPTIONS_TO_ADD=
      NR_OF_OPTIONS_TO_ADD=$(( RANDOM % MAX_NR_OF_RND_OPTS_TO_ADD + 1 ))
      OPTION_NAME=()
      for X in $(seq 1 ${NR_OF_OPTIONS_TO_ADD}); do
        OPTION_TO_ADD="$(shuf --random-source=/dev/urandom ${OPTIONS_INFILE} | head -n1)"
        if [ ${#OPTION_NAME[@]} -eq 0 ]; then
          OPTIONS_TO_ADD="$OPTIONS_TO_ADD $OPTION_TO_ADD"
          OPTION_NAME+=(${OPTION_TO_ADD%=*})
        elif [[ ! ${OPTION_NAME[@]} =~ ${OPTION_TO_ADD%=*} ]]; then
          OPTIONS_TO_ADD="$OPTIONS_TO_ADD $OPTION_TO_ADD"
          OPTION_NAME+=(${OPTION_TO_ADD%=*})
        fi
      done
      echoit "ADD_RANDOM_OPTIONS=1: adding mysqld option(s) ${OPTIONS_TO_ADD} to this run's MYEXTRA..."
      MYEXTRA="${MYEXTRA} ${OPTIONS_TO_ADD}"
    fi`
    
    Here, the code will enter the if loop when `ADD_RANDOM_OPTIONS` = 1. Later, we are setting MYEXTRA="${MYEXTRA} ${OPTIONS_TO_ADD}"
    
    This will populate MYEXTRA from the config file twice. We need to change this to
    MYEXTRA="${OPTIONS_TO_ADD}". This change needs to be done at many other places and this needs to be tested well

Reply:
Hi,
As discussed in DM: I have created a jira task with the issue description for more detailed look into the issue and closing this PR for now PSTRESS-156 .